src/web/wms/src/test/java/org/geoserver/wms/web/data/publish/WMSLayerConfigTest.java

```diff
@@ -167,4 +176,59 @@ public Component buildComponent(String id) {
 
         tester.assertModelValue("form:panel:defaultInterpolationMethod", WMSInterpolation.Bicubic);
     }
+
+    @Test
+    public void testWMSCascadeSettings() throws Exception {
+        MockHttpClient wms11Client = new MockHttpClient();
+        URL wms11BaseURL = new URL(TestHttpClientProvider.MOCKSERVER + "/wms11");
+        URL capsDocument =
+                WMSLayerConfigTest.class.getResource("/org/geoserver/wms/web/data/caps111.xml");
```

Comment:
If that file, which is added in this PR, is only used for this class, then it shoudl be added under `org.geoserver.wms.web.data.publish` and `this.getClass().getResource("caps111.xml")` used to retrieve it.

Reply:
understood

Reply:
kk